course02/project01/gavin/Asteroids/ship.cc

```diff
@@ -0,0 +1,84 @@
+#include "ship.h"
+#include "utils.h"
+#include "asteroid.h"
+#include <glm/gtc/random.hpp>
+#include <math.h>
```

Comment:
cmath

Reply:
Fixed in 76db252356e796ace0396de1cb5a68c9790e3d06